automation/src/main/java/org/wso2/carbon/testgrid/automation/executers/JmeterExecuter.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.testgrid.automation.executers;
+
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.testgrid.automation.exceptions.TestGridExecuteException;
+import org.wso2.carbon.testgrid.automation.executers.common.TestExecuter;
+import org.wso2.carbon.testgrid.common.Deployment;
+import org.wso2.carbon.testgrid.common.Host;
+import org.wso2.carbon.testgrid.common.Port;
+import org.wso2.carbon.testgrid.common.constants.TestGridConstants;
+import org.wso2.carbon.testgrid.utils.EnvVariableUtil;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * This class is responsible for performing the tasks related to execution of single jmeter solution.
+ */
+public class JmeterExecuter implements TestExecuter {
+
+    private static final Log log = LogFactory.getLog(JmeterExecuter.class);
+    private String jmterHome;
```

Comment:
For readability change this variable to jMeterHome. I think we should get this from an environment variable.